### DISCUSSION TITLE
Is it possible? Anonymous access without log in

### DISCUSSION THREAD

**Discussion by azlk** [filebrowser/filebrowser#1428] [Q&A]
I have an NFS share exported read only to /var/www/html. What I need is a simple file listing for ALL anonymous users without having to log in, like It made with nginx and fancyindex. Users need only to download files, nothing more. No delete, no cope-move etc.
Is it possible to do and how?

**Comment by ramiresviana** [comment]
Creating a shared folder allows users to have read-only access without authentication. You can also use the proxy authentication method to control the authentication using a header.

**Comment by azlk** [comment]
Thank you for your reply, but how can I make a shared folder with filebrowser? Can  not find such config set value and in web interface there is no way to set shared folder...
BTW I noticed this config set ` filebrowser config set --auth.method=noauth`, would it have the same result, I mean anonymous users' access without authentication? It is RO file system so I don't bother users not to delete/change anything there.

**Reply by ramiresviana** [reply]
In the current version, sharing a file/directory is done by clicking on the share icon on the header bar, the icon is displayed when a item is selected on the file listing. Using noauth authentication method allow users to access the app without loging in, but keep in mind that will allow access to any Filebrowser feature, such as hooks and commands that allow remote code execution.

**Reply by azlk** [reply]
ok, I see, thank you! I'll better use share link then

**Reply by albertsilliker** [reply]
In the json configuration file wich will be the noauth notation? I have tried "Auth method": "noauth" and it hasn't work
